src/apps/user-management/InviteUserDialog/InviteUserDialog.css

```diff
@@ -0,0 +1,134 @@
+.invite-user-dialog-overlay {
+  background-color: var(--black-a9);
+  position: fixed;
+  inset: 0;
+  animation: overlayShow 150ms cubic-bezier(0.16, 1, 0.3, 1);
+}
+
+.invite-user-dialog-content {
```

Comment:
Same here. Simply set the class to `dialog-content`, and the main stylesheet will kick in.